examples/passthrough/passthrough.bal

```diff
@@ -19,10 +20,11 @@ service passthrough on new http:Listener(9090) {
         if (clientResponse is http:Response) {
             // If the request was successful, an HTTP response is returned.
             // Here, the received response is forwarded to the client through the outbound endpoint.
-                var result = caller->respond(clientResponse);
-                if (result is error) {
-                   log:printError("Error sending response", err = result);
-                }
+            var result = caller->respond(clientResponse);
+            if (result is error) {
+                log:printError("Error sending response", err = result);
+                return result;
+            }
         } else if (clientResponse is error) {
```

Comment:
Can we just have the "else" here ?

Reply:
Yes we can. Since some of the BBEs, have used that pattern, I kept it as it is. All of them has to be refactored. I will update it here.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/12644/commits/13f67f9ec008a6b2f471aeb759c17b1483c3f75c